classes.py

```diff
@@ -0,0 +1,544 @@
+import json
+import math
+import os
+from urllib.parse import urlparse, parse_qs
+import csv
+
+import requests
+from bs4 import BeautifulSoup
+
+_CSV_DELIMITER = ';'
+_URLDOMAIN = "https://chess-results.com"
+_RATING_LIST_HEADER = 'Id_No;Id_CBX;Title;Name;Rtg_Nat;ClubName;Birthday;Sex;Fed;TotalNumGames;SumOpponRating' \
+                      ';TotalPoints'
+_MAX_NUM_GAMES_TEMP_RATING = 15
+_K_STARTING_NUM_GAMES = [(30, 0),  # grampo
+                         (25, _MAX_NUM_GAMES_TEMP_RATING),  # 15
+                         (15, 40),
+                         (10, 80)]
+_CURRENT_TOURNAMENT_SNR = 0
+
+
+class FexerjRatingCycle:
+
+    def __init__(self, tournaments_file, first_item, items_to_process, initial_rating_filepath):
+
```

Comment:
I'm not a big fan of these extra lines at the beginning of files or in the middle of functions, unless they separate logically-distinct blocks (like in the middle of __init__ now).

Reply:
Fixed.